### PR TITLE
fix to make the string being attached to  coloredLogStream colored again

### LINKED ISSUE #184: Output-colored.log is no longer coloring.

### PR DESCRIPTION
**Pull request by sahara-ooga**
Fixed a bug caused in PR #165 and #173 where strings for `output-colored.log` were no longer colored.

this fixes #184.

### CODE REVIEW

File: Sources/Public/Pretty.swift [hunk 1]
```diff
@@ -305,7 +305,14 @@ extension Pretty {
         option: Option
     ) {
         let plainString = printer(label, targets, separator, option)
-        let coloredString = printer(label, targets, separator, option)
+
+        let coloredOption: Option = {
+            var op = option
+            op.colored = true
+            return op
+        }()
```
Comment (YusukeHosonuma) [inline Sources/Public/Pretty.swift]: Wouldn't a code like the following suffice?

```suggestion
        let coloredOption = option
        coloredOption.colored = true
```

Comment (sahara-ooga) [reply Sources/Public/Pretty.swift]: Swift compiler says for the suggested above: 
> cannot assign to property: 'coloredOption' is a 'let' constant

`var` is ok:

```suggest
var coloredOption = option
coloredOption.colored = true
```

Comment (YusukeHosonuma) [reply Sources/Public/Pretty.swift]: Oops, I wrong.
Use `var` instead of `let` is corrected.

Comment (sahara-ooga) [reply Sources/Public/Pretty.swift]: @YusukeHosonuma 
I've refactored in 7582ec541c146644aad29090a88044299f0b839b.

**Review by YusukeHosonuma — APPROVED** [review]
👍

**Comment by sahara-ooga** [comment]
thx!

**Pull request merged**